### ISSUE TITLE
Admin Portal?

### ISSUE THREAD

**Issue by ecupaio** [hyweljohnllewellyn/carolsingershire#1]
I saw your [post](https://www.hywel.me/static/site/wordpress/2016/07/17/fast-free-static-website-with-wordpress-and-github-pages.html) on how to create a static wp site with gh-pages and I'm curious as to how users will update the site. 

Going to /wp-admin on the github domain and your static domain turns up the github 404 page. Am I reasonable to assume in order for users to update the site they will need to install git and create a github username (assuming they're a novice user), clone the repo, serve the repo locally (install MAMP), add/edit content, generate the static site, then push the changes to origin?

**Comment by hyweljohnllewellyn** [comment]
Hello.

To update the site once created:


1. Edit  site in wordpress
2. Re-generate static version using simply static Wordpress plugin (step 5
in my post)
3. Commit and push changes to github (step 6 in my post

The key is that the user edits the site using Wordpress , creates the
static version and then pushes the static files to GitHub.


The site is not edited directly on GitHub.

Does that help?
H

**Comment by ecupaio** [comment]
Yes and I executed that just fine on my own. Your instructions were very clear, but I'm wondering how do I set this up for a novice user (ie someone who doesn't have git installed locally)? As in, they could go to a subdomain admin portal (sub.example.com/wp-admin), publish edits to content, then have those edits push to the static site. 

I'm just wondering if there's a method for users to edit content online instead of locally.

**Comment by hyweljohnllewellyn** [comment]
Hi,
Quite a good online static editor is prose.io, see my post here:

https://www.hywel.me/2015/11/23/create-jekyll-static-site-posts-using-proseio.html

I have not reviewed other options for a while, though it sounds like you
may have some ideas.

I will look into it - but would be interested in more of your ideas and if
you have tried them.

Cheers.
